tools/article_checker/claude_retriever/client.py

```
@@ -147,7 +211,7 @@ def extract_statements(self, text: str, model: str, temperature: float = 0.0, ma
         return message.content[0].text
     
 
-    def retrieve(self, query: str, model: str, n_search_results_to_use: int = 3, stop_sequences: list[str] = [], max_tokens: int = 1000, max_searches_to_try: int = 5, temperature: float = 0.0) -> str:
+    def retrieve(self, query: str, model: str, n_search_results_to_use: int = 3, stop_sequences: list[str] = [], max_tokens: int = 1000, max_searches_to_try: int = 8, temperature: float = 0.0) -> str:
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)